project/BinaryIncompatibilities.scala

```diff
@@ -41,6 +41,8 @@ object BinaryIncompatibilities {
   )
 
   val Library = Seq(
+    // New methods in a native JS trait, not an issue
+    ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.scalajs.js.typedarray.TypedArrayStatic.*")
```

Comment:
The indentation should be 4 spaces here (see in `val Tools` above).

Reply:
Changed indentations of `Library` and `TestInterface` to `4`.